dags/process_elife_zip_dag.py

```diff
@@ -94,6 +96,19 @@ def prepare_jats_xml_for_libero(**context):
     return etree.tostring(xml)
 
 
+def send_put_request_to_service(**context):
+    return_value = get_return_value_from_previous_task(**context)
+    message = 'the previous task did not return data to send to %s' % SERVICE_URL
+    assert return_value is not None, message
+    response = requests.put(
+        SERVICE_URL,
```

Comment:
this being a PUT request, it is a dynamic URL containing the type and id of the item

Reply:
In the unstable environment, this will be http://unstable--api-gateway.libero.pub:8081/scholarly-articles/items/article1/versions/1 so the prefix `http://unstable--api-gateway.libero.pub:8081/scholarly-articles/` should be made configurable.
`article1` is the id of the article, extracted from the XML I guess.

Reply:
The URL was missing from the task/Issue. What is the structure of the url?

Reply:
This URL indicates that I need to extract the article ID and revision in order form the URL. Do you think I should move making the request to the previous task?